lib/wallet/http.js

```diff
@@ -183,6 +184,14 @@ class HTTP extends Server {
       this.logger.info('Successful auth for %s.', id);
     });
 
+    // Info
+    this.get('/', async (req, res) => {
+      res.json(200, {
+        version: pkg.version,
+        network: this.network.type
```

Comment:
Just these two values? You don't want to include `wdb.height` or number of wallets or anything like that?

Reply:
There could potentially be more information, yeah. The use case for this is to have a non-admin endpoint for an aliveness check. These fields, `version` and `network` match that of the node http info endpoint. It's not clear what other information could be useful here, yet. There could be a case to want to know the height, as relevant for rescans and etc. The number of wallets may require a potentially more expensive query, at this time. In effort to not introduce changes that would later lead to backwards incompatibility, other fields have been omitted for now.

Reply:
Sure see: https://github.com/handshake-org/hsd/pull/418

used for the rescan-progress use case in Bob wallet.